lib/src/query_component.dart

```diff
@@ -45,35 +49,31 @@ class QueryComponent implements OnInit {
   final QueryService _queryService;
   final ExcelDart _excel;
 
-  QueryType _queryType;
+  List<QueryType> queryTypeChoices = QueryType.values;
+  QueryType queryType;
 
-  String advertiserId;
-  String insertionOrderId;
+  String advertiserId = '164337';
```

Comment:
why these default values? i think it is best to leave it blank, it may not be ok to expose these ids publicly.

Reply:
My mistake, forgot to delete them when pushing :(